openedx/features/course_experience/views/course_outline.py

```diff
@@ -1,17 +1,19 @@
 """
 Views to show a course outline.
 """
+import re
+
 from django.template.context_processors import csrf
 from django.template.loader import render_to_string
 from opaque_keys.edx.keys import CourseKey
 from web_fragments.fragment import Fragment
 
-from courseware.courses import get_course_overview_with_access
+from completion import waffle as waffle
```

Comment:
You don't want `completion.waffle` anymore, you want `..waffle`

Reply:
You're reviewing too fast!